spec/unit/bowling_game_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'bowling_game'
+
+describe BowlingGame do
+  let(:game) { described_class.new }
+  let(:frame) { double :frame }
+
+  describe '#bowl' do
+    it 'can begin a new frame' do
+      allow(frame).to receive(:new).and_return(frame)
```

Comment:
I think we might want to return a different double - an double of an instance of the frame class. An example of this can be seen [here](https://github.com/makersacademy/skills-workshops/blob/f5b4801840fe07d26ff70341652dc81dcda12289/practicals/object_oriented_design/dependency_injection.md#intermezzo-of-classes)